.vscode/settings.json

```diff
@@ -1,37 +0,0 @@
-{
-    "editor.tabSize": 4,
-    "files.exclude": {
-        "**/.git": true,
-        "dev-env/lib/*": true,
-        "bazel-*/*": true,
-        ".bazel-cache/*": true,
-        "**/node_modules/*": true
-    },
-    "files.watcherExclude": {
-        "**/.stack-work": true,
-        "**/*.hi": true,
-        "**/*.dyn_hi": true,
-        "**/workspaces/*": true,
-        "**/.git": true,
-        "dev-env/lib/*": true,
-        "bazel-*/*": true,
```

Comment:
some of these excludes seem kind of useful. Is there a way to define this as a base rather than the actual settings file so they are set for all vscode users without the issue of users not being able to customize it?

Reply:
I couldn't find any other base file that vscode would look at for these settings, so I've now renamed it to `.vscode/settings.json.default` with a comment on top indicating it can be copied to `.vscode/settings.json` to enable the settings